Audio/MenuLoop.mp3.import

```diff
@@ -0,0 +1,19 @@
+[remap]
+
+importer="mp3"
+type="AudioStreamMP3"
+uid="uid://fdeyxf6qyqp2"
+path="res://.godot/imported/MenuLoop.mp3-838095eae95fca79c0ff7f1894480925.mp3str"
+
+[deps]
+
+source_file="res://Audio/MenuLoop.mp3"
+dest_files=["res://.godot/imported/MenuLoop.mp3-838095eae95fca79c0ff7f1894480925.mp3str"]
+
+[params]
+
+loop=true
+loop_offset=32.0
```

Comment:
Unfortunately this doesn't seem to work in the web export: the music loops from the beginning. It might work if the node were set to "sample" playback mode, per https://docs.godotengine.org/en/latest/tutorials/export/exporting_for_web.html#audio-playback, though looping is not one of the features that is not documented as being unsupported by the "stream" mode.

It would be easier to change the music to loop cleanly from start to finish. The reverb & delay effects on the harp chords make this difficult because the last chords of the end of the phrase can still be heard reverberating at the start of the next repetition.